.changeset/proud-ghosts-learn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": patch
```

Comment:
This seems to just be removing an unused type, so I put it as patch, but it does touch the UserInput type and the Props type, so wasn't sure if that meant it should be minor. Leaning towards `patch`, but wanted to confirm :)